dataconnect/dataconnect/movie-connector/connector.yaml

```diff
@@ -0,0 +1,6 @@
+connectorId: movie-connector
+authMode: PUBLIC
+generate:
+  javascriptSdk:
+    package: "@movie/sdk"
+    outputDir: ../../app/lib/dataconnect-sdk
```

Comment:
Is there a reason why we're using the relative path here instead of the `packageJsonDir`?